### ISSUE TITLE
Upgrade to API version 4?

### ISSUE THREAD

**Issue by Xunnamius** [clns/gitlab-cli#13]
Hi. This is an excellent tool, thanks for this! Are there any plans to upgrade this to use the V4 version of the GitLab API? V3 has been deprecated as of GitLab 9.5 (we're past version 10 now).

Update for those in search of a solution: since I wasn't able to find any other tools that can propagate my global admin labels across my instance that also work with the latest version of GitLab, I made one myself: https://www.npmjs.com/package/gitlab-fix-labels. It also addresses #11.

Cheers!

**Comment by analog-cbarber** [comment]
Yes, sadly this tool no longer works for me since we upgraded our server. It probably wouldn't be that hard to switch to v4, but I expect most users to just switch to another tool.

**Comment by analog-cbarber** [comment]
It doesn't seem that Xunnamius's project can copy labels from one project to another. It looks like it can only copy the admin labels to a project.
